Vagrantfile

```diff
@@ -1,26 +1,204 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 
+CONTAINERD_2_YOUKI_GO_VERSION = "1.20.12"
```

Comment:
Nit: reason to pick this version? Asking since there are newer version. (Just curious, no preference here).

Reply:
Thanks for the question! I just migrated the Go version that was originally used:
https://github.com/youki-dev/youki/blob/4ab9742bcc37155e0874162ec3f5fb04121e96ce/Vagrantfile.containerd2youki#L4